src/main.js

```diff
@@ -16,6 +16,13 @@ let createGitIgnoreFile = async (options) => {
   return;
 }
 
+let generateEnvFile = async (options) => {
```

Comment:
Ok, the code works @HritikR Nice work. Two things though. Please help change the word **generate** to **create** in the function name for the .env files (so that the function name doesn't look so different from what the gitignore function does). Also add letter "s" to the end of the function name for the .env files since we are creating more than one .env files.